src/Runner.Listener/Runner.cs

```diff
@@ -50,7 +51,8 @@ public override void Initialize(IHostContext hostContext)
         {
             base.Initialize(hostContext);
             _term = HostContext.GetService<ITerminal>();
-            _acquireJobThrottler = HostContext.CreateService<IErrorThrottler>();
```

Comment:
we should always use `GetService` instead of `CreateService` for singleton service.

Reply:
This was originally intended not to be a singleton. Why switch to singleton?

Reply:
> The error throttler helps us back off when encountering successive, non-retriable errors from /acquirejob.

Reply:
i didn't find any place we use `IErrorThrottler` outside of the Runner, so i thought we only need one.
I guess, if the original idea is to create `IErrorThrottler` as needed, then `CreateService` make more sense.

Reply:
i changed it back for now.